subxt/examples/legacy_rpc.rs

```diff
@@ -0,0 +1,96 @@
+pub mod polkadot {}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Create both an OnlineClient and a LegacyRpcClient, configured to talk to Polkadot
+    // nodes with the same RpcClient using the default URL
+    let rpc_client = RpcClient::from_url("ws://127.0.0.1:9944").await?;
+    let api = OnlineClient::<PolkadotConfig>::from_rpc_client(rpc_client.clone()).await?;
+    let rpc = LegacyRpcMethods::<PolkadotConfig>::new(rpc_client);
+
+    let alice = dev::alice().public_key().to_account_id();
+
+    // Some example destinations accounts and amounts
+    let transactions = vec![
+        (dev::bob().public_key(), 10000000),
+        (dev::charlie().public_key(), 12000000),
+        (dev::ferdie().public_key(), 30000000),
+        (dev::eve().public_key(), 40000000),
+        (dev::dave().public_key(), 50000000),
+        (dev::bob().public_key(), 1000000),
+        (dev::charlie().public_key(), 1200000),
+        (dev::ferdie().public_key(), 3000000),
+        (dev::eve().public_key(), 4000000),
+        (dev::dave().public_key(), 50000000),
+        (dev::bob().public_key(), 50000000),
+        (dev::charlie().public_key(), 22000000),
+        (dev::ferdie().public_key(), 130000000),
+        (dev::eve().public_key(), 40003000),
+        (dev::dave().public_key(), 150000000),
+    ];
+
+    println!(
+        "📛 System Name: {:?}\n🩺 Health: {:?}\n🖫 Properties: {:?}\n🔗 Chain: {:?}\n",
+        rpc.system_name().await?,
+        rpc.system_health().await?,
+        rpc.system_properties().await?,
+        rpc.system_chain().await?
+    );
+
+    for (dest, amount) in transactions {
+        let nonce = rpc.system_account_next_index(&alice).await?;
+
+        // Extrinsics are mortal, only valid for a limited number of blocks
+        // from the current best
+        let extrinsic_parameters = rpc
+            .chain_get_header(None)
+            .await?
+            .map(|header| {
+                DefaultExtrinsicParamsBuilder::default()
+                    .mortal(&header, 8)
+                    .build()
+            })
+            .ok_or(Error::UnexpectedNone)?;
+        let balance_transfer = polkadot::tx()
+            .balances()
+            .transfer_allow_death(dest.into(), amount);
+        let extrinsic = api.tx().create_signed_with_nonce(
+            &balance_transfer,
+            &dev::alice(),
+            nonce,
+            extrinsic_parameters,
+        )?;
+        println!("Submitting {:?} with Nonce:{nonce}", extrinsic.hash());
+
+        // Submitting without tracking the progress of the extrinsics
+        let _ = extrinsic.submit().await?;
+
+        // Account nonce will return the next nonce based on finalized extrinsics but
+        // RPC call System.account_next_index will adjust for transactions already in
+        // the pool
+        println!(
+            "AccountNonce: {:?} System.account_next_index: {:?}",
```

Comment:
IIUC, without the 4 seconds sleep we'd eventually after running a few times have different values those 2 methods?

Reply:
@lexnv even wihtout the delay `api.tx().account_nonce(&alice)` will lag